notifications/rest/users/create-user/create-user.5.x.cs

```diff
@@ -0,0 +1,25 @@
+// Download the twilio-csharp library from www.twilio.com/docs/libraries/csharp#installation
+using System;
+using System.Collections.Generic;
+using Twilio;
+using Twilio.Rest.Notify.V1.Service;
+
+public class Example
+{
+    public static void Main(string[] args)
+    {
+        // Find your Account SID and Auth Token at twilio.com/console
+        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        const string authToken = "your_auth_token";
+
+        const string serviceSid = "ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+
+        TwilioClient.Init(accountSid, authToken);
+
+        NotifyUserResource user = NotifyUserResource.create(
```

Comment:
I believe the name of the `NotifyUserResource` class might be `UserResource instead` since the class name in the `user.json` in _api-definitions_ repo is _user_.